Fix list highlight on Customer Review Select #2646

fragm3 opened the pull request:
Fixes #2646, #2645, #2648, #2651

Changes: 
- [x] Fix list highlight on Customer Review Select
- [x] Added list highlight on hover
- [x] Scroll to top on metric change
- [x] Fix settings height overflow
- [x] Fix browseskill add url params on filter apply(Half issue)
- [x] Fixed avatar circular image

Demo Link: https://pr-2647-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

<img width="369" alt="Screenshot 2019-07-21 at 5 02 48 PM" src="https://user-images.githubusercontent.com/18073126/61590596-6444e380-abd9-11e9-9706-ed7050d4bdf0.png">

Fixed this:

<img width="441" alt="Screenshot 2019-07-21 at 5 08 19 PM" src="https://user-images.githubusercontent.com/18073126/61590654-34e2a680-abda-11e9-935d-2f3f3e4ee568.png">

<img width="1126" alt="Screenshot 2019-07-21 at 8 48 08 PM" src="https://user-images.githubusercontent.com/18073126/61598061-c59aa000-ac35-11e9-9c03-3158515d9569.png">

<img width="553" alt="Screenshot 2019-07-21 at 8 48 41 PM" src="https://user-images.githubusercontent.com/18073126/61598062-c59aa000-ac35-11e9-9e28-da588a8f0158.png">

akshatnitd commented on src/components/cms/BrowseSkill/BrowseSkill.js:
```diff
@@ -410,12 +409,14 @@ class BrowseSkill extends React.Component {
     this.props.actions.getMetricsSkills({
       languageValue: this.props.languageValue,
     });
+    scrollAnimation(document.documentElement, 0, 200, 'vertical');
```
How about implementing in the `withTracker` HOC in your other PR?

fragm3 replied:
Oh, nice idea. But we would require inline scroll calls as well for metrics change on browseskill

akshatnitd replied:
To handle that, we should actually add the url param on metric change to the url as well.

fragm3 replied:
Okay working on it, created issue https://github.com/fossasia/susi.ai/issues/2651

fragm3 commented:
@akshatnitd done, https://github.com/fossasia/susi.ai/pull/2634#discussion_r305622193

fragm3 commented:
@akshatnitd I have integrated query string in browse skill. I will pick up order maintain in the next PR.

**Note**: I had implemented queryString for **pages, entries per page** as well, but didn't won't work well as we splitting the skills on frontend instead of backend. So for now, removed that code.

fragm3 commented:
@akshatnitd fixed the link
